quartz/plugins/index.ts

```diff
@@ -18,6 +19,28 @@ export function getStaticResourcesFromPlugins(ctx: BuildCtx) {
     }
   }
 
+  const { theme } = ctx.cfg.configuration
+  if (theme.fontOrigin === "googleFonts" && theme.cdnCaching) {
+    staticResources.css.push(googleFontHref(theme))
+  }
```

Comment:
I would liked to have kept this in ComponentResources but I got an error if I did `componentResources.css.push(googleFontHref(theme))`

```
Failed to emit from plugin `ComponentResources`: Invalid token in pseudo element: Delim('/')
```
from lightningcss transform.

Reply:
yeah let's figure out a way to keep it in ComponentResources

Reply:
Moved to Head.tsx, next to the other google fonts <link> elements